assets/static/style.css

```diff
@@ -21,7 +21,7 @@
 
 
 body {
-    font-family: mono;
+    font-family: Courier;
```

Comment:
Make this say "Courier, mono;"

This means that if they have Courier, it will show that, but if they don't it will fall back to whatever monospace they have.